app/src/main/java/de/tum/in/tumcampusapp/component/ui/transportation/DepartureView.kt

```diff
@@ -0,0 +1,185 @@
+package de.tum.`in`.tumcampusapp.component.ui.transportation
+
+import android.animation.ValueAnimator
+import android.content.Context
+import android.content.res.ColorStateList
+import android.graphics.Color
+import android.os.Handler
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.view.animation.AccelerateInterpolator
+import android.view.animation.AnimationUtils
+import android.widget.LinearLayout
+import android.widget.TextSwitcher
+import android.widget.TextView
+import de.tum.`in`.tumcampusapp.R
+import org.joda.time.DateTime
+import org.joda.time.Seconds
+import java.util.*
+
+/**
+ * Custom view that shows a departure.
+ * Holds an icon of the subway public transfer line, the line name and an animated
+ * automatically down counting departure time
+ */
+class DepartureView
+/**
+ * Constructor for DepartureView
+ *
+ * @param context Context
+ * @param big     Whether the departure should use a thin or a big line
+ */
+@JvmOverloads constructor(context: Context, private val big: Boolean = false) : LinearLayout(context) {
+    private val symbolView: TextView by lazy { findViewById<TextView>(R.id.line_symbol) }
+    private val lineView: TextView by lazy { findViewById<TextView>(R.id.nameTextView) }
+    private val timeSwitcher: TextSwitcher by lazy { findViewById<TextSwitcher>(R.id.line_switcher) }
+    private val mHandler: Handler
```

Comment:
The convention of prefixing members with `m` is no longer recommended. I suggest to adapt the code here and in other files of this PR.